app/models/tagging.rb

```diff
@@ -42,4 +43,10 @@ def update_search
     # in multiple fandoms.
     tagger.enqueue_to_index if tagger.taggings_count < ArchiveConfig.TAGGINGS_COUNT_REINDEX_LIMIT
   end
+
+  def collection_tag_limit
+    return unless taggable.class == Collection && taggable.tags.count >= ArchiveConfig.COLLECTION_TAG_MAX
+
+    errors.add(:tags, "You have reached the tag limit")
```

Comment:
Can you make this a translatable string and include the limit? Similar to:

https://github.com/otwcode/otwarchive/blob/bedbbef8c43585d97b5a56062355921e4c8f5fa7/app/models/favorite_tag.rb#L14-L20